network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/extensions/RemoteReactivePowerControlImpl.java

```diff
@@ -65,4 +74,43 @@ public Terminal getRegulatingTerminal() {
         }
         return null;
     }
+
+    @Override
+    public RemoteReactivePowerControl setRegulatingTerminal(Terminal regulatingTerminal) {
```

Comment:
I think if it has a regulatingTerminal it should have a regulatingPoint

Reply:
There is no use of the RegulatingPoint class in powsybl-core in the RemoteReactivePowerControl extension ...
And there is no fallback between local and remote regulation in this case ...